### ISSUE TITLE
[dontorrent] (configuring) <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN"><html><head><title>301 Moved Permanently</title></head><body><h1>Moved Permanently</h1><p>The document has moved <a href="https://dontorrent.contact/ultimos">here</a>.</p></body></html>

### ISSUE THREAD

**Issue by vmasmitja** [Jackett/Jackett#14885]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Browser: Brave Browser
Browser version: 1.58.137
Browser version Chromium: 117.0.5938.153 (Build oficial) (arm64)
OS: HASSIO
HA Core: 2023.11.3
HA Supervisor: 2023.11.6
HA Operating System: 11.2
HA Frontend: 20231030.2
Jackett Version v0.21.1289
NO PROXY
```

### Description

On indexer add DonTorrent, fail.

### Logged Error Messages

```text
[v0.21.1289.0] Jackett.Common.ExceptionWithConfigData: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://dontorrent.contact/ultimos">here</a>.</p>
</body></html>

   at Jackett.Common.Indexers.DonTorrent.PerformQueryNewest(TorznabQuery query) in ./Jackett.Common/Indexers/DonTorrent.cs:line 190
   at Jackett.Common.Indexers.DonTorrent.PerformQuery(TorznabQuery query) in ./Jackett.Common/Indexers/DonTorrent.cs:line 149
   at Jackett.Common.Indexers.DonTorrent.ApplyConfiguration(JToken configJson) in ./Jackett.Common/Indexers/DonTorrent.cs:line 133
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in ./Jackett.Server/Controllers/IndexerApiController.cs:line 97
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in ./Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61
```

### Screenshots

![screenshot-172 30 148 33_9117-2023 12 06-12_23_14](https://github.com/Jackett/Jackett/assets/12512254/86057381-3a65-48d0-a8d8-e5f1c43651c2)

**Comment by garfield69** [comment]
Your Jackett version is out of date.
Upgrade to 0.21.1315 to resolve your issue.

**Comment by Uklosk** [comment]
I have the same issue with version v0.21.1396

**Comment by garfield69** [comment]
>I have the same issue with version v0.21.1396
Really?
If you are on 0.21.1396 then when you use your Jackett dashboard to edit the dontorrent config your sitelink should already be set to `https://dontorrent.cymru/`, which I have tested just now and works for me.
If yours is not that address, then double check your version and upgrade, or replace whatever the sitelink is now with the .cymru that one I provided above.
